common/test/acceptance/tests/studio/test_studio_container.py

```diff
@@ -413,6 +422,17 @@ def select_and_verify_saved(self, component, partition_label, groups=[]):
         self.verify_selected_groups(visibility_editor, groups)
         visibility_editor.save()
 
+    def select_and_verify_unit_group_access(self, unit, partition_label, groups=[]):
+        """
+        Edit the visibility of an xblock on the unit page and
+        verify that the edit persists. Note that `groups`
+        are labels which should be clicked, but are not necessarily checked.
+        """
+        unit_access_editor = self.edit_unit_visibility(unit)
+        unit_access_editor.select_groups_in_partition_scheme(partition_label, groups)
+        unit_access_editor = self.edit_unit_visibility(unit)
```

Comment:
The name of this method says "_and_verify". Did you intend to verify the selection, like was done in select_and_verify_saved?